NeoForgeApi/src/main/java/mezz/jei/api/neoforge/NeoForgeTypes.java

```diff
@@ -0,0 +1,35 @@
+package mezz.jei.api.neoforge;
+
+import mezz.jei.api.ingredients.IIngredientType;
+import mezz.jei.api.ingredients.IIngredientTypeWithSubtypes;
+import net.minecraft.world.level.material.Fluid;
+import net.neoforged.neoforge.fluids.FluidStack;
+
+/**
+ * Built-in {@link IIngredientType} for NeoForge Minecraft.
+ *
+ * @since 9.6.0
+ */
+public final class NeoForgeTypes {
+    /**
+     * @since 9.7.0
```

Comment:
have to be careful to update `@since` tags in the api package to match the new version